docs/api/browser-window.md

```diff
@@ -202,7 +202,7 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
     than screen. Default is `false`.
   * `backgroundColor` String (optional) - Window's background color as a hexadecimal value,
     like `#66CD00` or `#FFF` or `#80FFFFFF` (alpha is supported). Default is
-    `#FFF` (white).
+    `#FFF` (white). If `transparent` is set to `true`, only transparent (`#00000000`) or opaque (`#FFFFFFFF`) are respected.
```

Comment:
This isn't correct. _Any_ color is allowed as long as the alpha bits are FF or 00.

When `transparent` is set to `true`, the color's alpha bits must all-on or all-off. For example `#80FFFFFF` would not be OK but `#00FFFFFF` and `#FFFFFFFF` would.

Reply:
ohhh i see! editing now 😸